politeiawww/templates.go

```diff
@@ -28,3 +28,10 @@ const templateUpdateUserKeyEmailRaw = `
 <span style="font-weight: bold">{{.Email}}</span> on Politeia. If you did not perform
 this action, please contact Politeia administrators.</div>
 `
+
+const templateUserLockedResetPasswordRaw = `
+<div>Your account was locked due to many login attempts</div>
+<div>You need to reset your password in order to unlock your account</div>
+<div>Click the link below to continue resetting your password:</div>
+<div style="margin: 20px 0 0 10px"><a href="{{.Link}}">{{.Link}}</a></div>
```

Comment:
This link can point to the UI `/password` route, maybe with the email already populated in the field so he just has to click "Reset password"?

Reply:
Fixed.
This PR is ready for another review. I needed to make some change at gui in order to auto populate the email. Here is the PR: https://github.com/decred/politeiagui/pull/476